### DISCUSSION TITLE
GPU out of memory while training HighResNet but works fine when training Unet.

### DISCUSSION THREAD

**Discussion by Ric1779** [Project-MONAI/MONAI#6065] [Q&A]
I'm trying to train highresnet but there's an error stating that the gpu is out of memory, but I can train the Unet model. I tried training the highresnet without the residual blocks, i.e. only with three conv layers, but still have the same problem.
![Screenshot from 2023-02-25 19-09-00](https://user-images.githubusercontent.com/29515662/221372843-8bc23abb-b323-473e-aeb2-0a0df6b55f78.png)

![Screenshot from 2023-02-25 19-10-49](https://user-images.githubusercontent.com/29515662/221372905-b3072dfc-106c-4890-84fc-f5a0f0c9b60c.png)

**Comment by KumoLiu** [comment]
Hi @Ric1779, I noticed you are using jupyter, did you try to restart the kernel before your training? Sometimes jupyter can't automatically releasing the cache. Thanks!

**Reply by Ric1779** [reply]
Yes. I've tried it. But it does not work.

**Reply by KumoLiu** [reply]
Hi @Ric1779, could you please share a piece of code which can help reproduce the issue, then I  could take a deep look at it, thanks!
